react-hot-button/my-app/src/HotButton.tsx

```diff
@@ -0,0 +1,15 @@
+import './App.css';
+
+type Props = {
+  onButtonClick: () => void;
+
+  btnLvlClass: string;
```

Comment:
It's best to compute the classname inside this component so that the parent doesn't have to know what class name(s) are used.